File: remote-task/src/main/java/org/littlewings/infinispan/task/entity/Book.java
package org.littlewings.infinispan.task.entity;

import java.io.Serializable;

public class Book implements Serializable {
    private static final long serialVersionUID = 1L;

    private String isbn;
    private String title;
    private int price;

    public Book(String isbn, String title, int price) {
        this.isbn = isbn;
        this.title = title;
        this.price = price;
    }

    public String getIsbn() {
        return isbn;
    }

    public String getTitle() {
        return title;
    }

    public int getPrice() {
        return price;
    }
}

File: embedded-data-containers/src/test/scala/org/littlewings/infinispan/datacontainers/Book.scala
package org.littlewings.infinispan.datacontainers

object Book {
  def apply(isbn: String, title: String, price: Int): Book =
    new Book(isbn, title, price)
}

@SerialVersionUID(1L)
class Book(val isbn: String, val title: String, val price: Int) extends Serializable
